composer.json

```diff
@@ -41,7 +18,7 @@
     "wpackagist-plugin/pantheon-advanced-page-cache": "^0.1.4",
     "wpackagist-plugin/wp-cfm": "^1.4.5",
     "wpackagist-theme/twentyseventeen": "^1.1",
-    "pantheon-systems/wordpress": "^4.8",
+    "pantheon-systems/wordpress-composer": "dev-master",
```

Comment:
You make some good points @greg-1-anderson. I'll concede in part because we are committing `composer.lock` and will need to update here regardless.